### PR TITLE
🙏🙏Added Materialize Integration 

### LINKED ISSUE #3568: [Integration] Implement handler for Materialize
Add the support for https://materialize.com/. To start check out the [Build new integration docs](https://docs.mindsdb.com/contribute/integrations/)

## :loudspeaker: Additional rewards :trophy:

Each integration brings :five:  points and entry into the draw for a  :point_down: :

-  :computer:  [Razer Blade 15 Laptop](https://www.razer.com/hard-bundles/Razer-Blade-15-Bundle-V5-Essential/RZHB-220912-11)
- :headphones:  10 cool audio gadgets 
- :shirt: :bear: MindsDB Swag
:information_source: For more info check out https://mindsdb.com/hacktoberfest/ :point_left: 

To participate in the prize draw, you need to complete this form with your contact details and links to all your merged PRs or no-code deliverables. Please submit the form once with all your contributions listed.

### PR DESCRIPTION
**Pull request by parthiv11**
Fixes #3568 

## Please describe what changes you made, in as much detail as possible
  - Implemented Materialize Database .
  - Please check it and suggest changes, if Required 

mindsdb

### CODE REVIEW

**Comment by parthiv11** [comment]
For testing either you can apply for [early access](https://materialize.com/register/) or [build from source](https://github.com/MaterializeInc/materialize/tree/main/doc/developer)

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]
Please make sure the branch is up to date so the tests will not fail

**Comment by parthiv11** [comment]
@ZoranPandovski  it's syned now .

You can merge it n0w

File: mindsdb/integrations/handlers/materialize_handler/materialize_handler.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+from mindsdb.integrations.libs.const import HANDLER_CONNECTION_ARG_TYPE as ARG_TYPE
+from collections import OrderedDict
+
+
+from ..yugabyte_handler import Handler as YugaByteHandler
+
+
+class MaterializeHandler(YugaByteHandler):
```
Comment (ZoranPandovski) [inline mindsdb/integrations/handlers/materialize_handler/materialize_handler.py:8]: Let's extend the PostgreSQL handler that is most wildly used and better maintained

File: mindsdb/integrations/handlers/materialize_handler/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+This is the implementation of the Materialize handler for MindsDB.
+
+## Materialize
+Materialize is a new storage engine for PostgreSQL, bringing a modern approach to database capacity, capabilities and performance to the world's most-loved database platform.
+
+Materialize consists of an extension, building on the innovative table access method framework and other standard Postgres extension interfaces. By extending and enhancing the current table access methods, Materialize opens the door to a future of more powerful storage models that are optimized for cloud and modern hardware architectures.
+## Implementation
+
+This handler was implemented by extending postres connector.
```
Comment (ZoranPandovski) [inline mindsdb/integrations/handlers/materialize_handler/README.md:11]: Minor typo in `postres`

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]

**Comment by parthiv11** [comment]
@ZoranPandovski  check the changes

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**